bug/op_edit_comment.go

```diff
@@ -51,7 +52,7 @@ func (op *EditCommentOperation) Apply(snapshot *Snapshot) {
 	}
 
 	comment := Comment{
-		id:       op.Target,
+		id:       entity.CombineIds(snapshot.Id(), op.Id()),
```

Comment:
@MichaelMure: Is that the correct Id for the new `Comment`?

Reply:
That's incorrect actually. A comment Id is the same Id as the operation that created it (only combined with the Bug Id).

Here you combined the Bug Id with the *edit* operation Id.

Reply:
I'll fix it and try to make things a bit more clear in the code.